src/components/common/YourCompany/YourCompany.vue

```diff
@@ -259,8 +262,9 @@
     <v-divider class="mx-4 my-1" />
 
     <!-- Office addresses -->
-    <div class="section-container">
+    <div id="business-addresses" class="section-container" :class="{'invalid-section': invalidAddressSection}">
       <OfficeAddresses
+        :invalidSection="invalidAddressSection"
```

Comment:
I think you can combine the div and OfficeAddresses here also.

Reply:
OK, nvm.

Reply:
Looking into this though still, it seems like the same issue with the Nature of Business wrapper. 
Somehow rearranges the selectors, will try to find out how!

Reply:
I think what Vue is doing is copying the shared component into the `<NatureOfBusiness>` element. So properties are merged.

Reply:
Interesting!

Reply:
You were right! Conflict of Id's, removed the parent wrapper Id's and applied the component ID for the validations